tc_generator/tc_gen.py

```diff
@@ -33,18 +33,25 @@
 
 def generate(choice, i):
     try:
-        os.system('%s < input/input%02d.txt > output/output%02d.txt' %
-                  (LANGS[choice - 1]['command'], i, i))
-
+        if platform.system()=='Windows':
```

Comment:
We could have just compared the first letter, also have you tested it.
Is it `Windows` or `windows`